lib/node/http.js

```diff
@@ -188,14 +188,27 @@ class HTTP extends Server {
     });
 
     // Bulk read UTXOs
+    // TODO(boymanjor): Deprecate this endpoint
```

Comment:
This is not related to the wallets, this is part of indexer API.

Reply:
@nodar-chkuaselidze We had discussed having the wallet provide the functionality of these endpoints: "Importing a single address (and multiple) as a watch only wallet, would provide the balance and pagination of history, which is another option." https://github.com/bcoin-org/bcoin/issues/589#issuecomment-439115733

Reply:
okay, Do we deprecate existing indexers as well ?

Reply:
Yeah, without these endpoints the `addrindex` isn't used and can be deprecated too.

However, `txindex` would remain as it doesn't have that issue, and the upcoming `filterindex` can more efficiently support not only nuentrino nodes, but also be useful for speeding up wallet rescans/resets , and additionally provide a way for wallets to rescan/reset on pruned nodes, selectively querying only the blocks necessary for peers.